notebooks/optimal-stopping-point/README.md

```diff
@@ -0,0 +1,31 @@
+# Optimal Stopping Point Prediction
+
+The aim of this ML problem is to predict an optimal stopping point for CI tests based on their test duration (runtimes). We perform an initial data analysis as well as feature engineering on the testgrid data. Furthermore, we also calculated the optimal stopping point by identifying the distribution of the test_duration values for different CI tests and comparing the distributions of passing/failing tests.
```

Comment:
change `test_duration` to "test duration"?
maybe mention it as "comparing the distributions between the passing and failing tests"?